common/src/app/common/pages/changes_builder.cljc

```diff
@@ -276,6 +276,21 @@
          (update :undo-changes #(reduce mk-undo-change % shapes))
          (apply-changes-local)))))
 
+
+(defn changed-attrs
```

Comment:
Looks unused.

Reply:
I know, I left it because I think could be used in the future